batch/batch/cloud/gcp/instance_config.py

```diff
@@ -63,8 +68,13 @@ def create(
         ]
 
         accelerator_family = machine_family_to_gpu(machine_type_parts.machine_family)
+
         if accelerator_family:
-            resources.append(GCPAcceleratorResource.create(product_versions, accelerator_family, preemptible, region))
+            num_gpus = machine_type_to_gpu_num(machine_type)
+            assert num_gpus
```

Comment:
`machine_type_to_gpu_num` always returns a positive number so there's no need to `assert num_gpus`